docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -92,17 +92,18 @@ DriverConfigurationProfiles.addOrReplaceProfile(
+| Log Query Connection Plugin                                                                 | `logQuery`              | Tracks and logs the SQL statements to be executed. Sometimes SQL statements are not passed directly to the JDBC method as a parameter, such as [executeBatch()](https://docs.oracle.com/javase/8/docs/api/java/sql/Statement.html#executeBatch--). Users can set `enhancedLogQueryEnabled` to `true`, allowing the JDBC Wrapper to obtain SQL statements via Java Reflection. <br><br> :warning:**Note:** Enabling Java Reflection may cause a performance degradation.                                                   |
+| [IAM Authentication Connection Plugin](./using-plugins/UsingTheIamAuthenticationPlugin.md)  | `iam`                   | Enables users to connect to their Amazon Aurora clusters using AWS Identity and Access Management (IAM).                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| [AWS Secrets Manager Connection Plugin](./using-plugins/UsingTheAwsSecretsManagerPlugin.md) | `awsSecretsManager`     | Enables fetching database credentials from the AWS Secrets Manager service.                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Aurora Stale DNS Plugin | `auroraStaleDns`          | Prevents opening a new connection to an old writer node when DNS has stale records and iut's not yet updated after a recent failover event. <br><br> :warning:**Note:** Contrary to `failover` plugin, `auroraStaleDns` plugin doesn't implement failover support itself. It helps to eliminate opening wrong connections to an old writer node after cluster failover is completed. <br><br> :warning:**Note:** This logic is already included in `failover` plugin so you can omit using both plugins at the same time. |
```

Comment:
typo: `iut's` -> "Prevents incorrectly opening a new connection to an old writer node when DNS records have not yet updated after a recent failover event".